br/pkg/storage/s3.go

```diff
@@ -292,7 +296,11 @@ func createOssRAMCred() (*credentials.Credentials, error) {
 		return nil, errors.Annotate(err, "Alibaba RAM Provider Retrieve")
 	}
 	ncred := cred.(*alicred.StsTokenCredential)
-	return credentials.NewStaticCredentials(ncred.AccessKeyId, ncred.AccessKeySecret, ncred.AccessKeyStsToken), nil
+	return credentials.NewChainCredentials([]credentials.Provider{
```

Comment:
Can we add the `aliproviders.NewInstanceMetadataProvider()` to the chain directly?